service-app/internal/api/index_controller.go

```diff
@@ -230,24 +230,28 @@ func (c *IndexController) ingestHandler(w http.ResponseWriter, r *http.Request)
 	}
 }
 
-func getPublicError(err error, uid string) string {
+func getPublicError(err error, uid string) (int, string) {
 	var clientError sirius.Error
 	if !errors.As(err, &clientError) {
-		return ""
+		return 500, ""
```

Comment:
I think it would be less confusing to have these return the "Failed to persist document to Sirius" message, then remove the `if`